engine/text/Text.hpp

```diff
@@ -166,6 +166,18 @@ class Text {
         mPropertiesUpdated = true;
     }
 
+    /**
+    * Sets the colour of the background.
+    *
+    * @param colour    The new colour.
+    */
+    void setBackgroundColour(SDL_Color colour) {
+      mBackColour = colour;
+      mHasBackground = true;
```

Comment:
Unnecessary bool, background colour can be made transparent instead of using this bool